### PR TITLE
adds celldemo_FindMaxima_makeSpots.groovy

### PR DESCRIPTION
**Pull request by romainGuiet**

### CODE REVIEW

File: scripts/celldemo_FindMaxima_makeSpots.groovy [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+rm.runCommand(c3_imp,"Measure");
+
+
+/** B - create Imaris spots from the Results Table
+ */
+// here all the spots will have same dimensions (you can also make a List of points)
+radiusXYZ = new Point3D()
+radiusXYZ.setX(xRad_spot as double )
+radiusXYZ.setY(yRad_spot as double)
+radiusXYZ.setZ(zRad_spot as double )
```
Comment (NicoKiaru) [inline scripts/celldemo_FindMaxima_makeSpots.groovy:71]: Why are these lines necessary ? I don't get it

File: scripts/celldemo_FindMaxima_makeSpots.groovy [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+coordinates = (0..<rt.getCounter()).collect { it ->
+ 	pt = new Point3D()
+ 	pt.setX( xS[it] as double )
+ 	pt.setY( yS[it] as double )
+ 	pt.setZ( zS[it] as double )
+ 	return pt
+}
+// Finally create the ImarisSpots 
+spots = EasyXT.Spots.create(coordinates , radiusXYZ, 0)// single timepoint -> 0
```
Comment (NicoKiaru) [inline scripts/celldemo_FindMaxima_makeSpots.groovy:92]: I still don't get it

**Review by NicoKiaru — COMMENTED** [review]
Looks nice, few one important thing at the end that I did not get... (see comment)

File: scripts/celldemo_FindMaxima_makeSpots.groovy [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+ */
+
+// start-up cleaning
+rt.reset()
+rt.updateResults()
+rm.reset();
+IJ.run("Close All", "");
+EasyXT.Utils.connectToImaris()
+
+// imports
```
[the remark this answers is not in the document]
Comment (romainGuiet) [reply scripts/celldemo_FindMaxima_makeSpots.groovy]: fixed in 17a9d97d8a7e28c73eff34f323df17e874fa899e

File: scripts/celldemo_FindMaxima_makeSpots.groovy [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+if (showImages) c3_imp.show()
+
+/** A - make use of Fiji to detect spots
+ */
+
+// We'll just apply a Median 3D to have less point
+IJ.run(c3_imp, "Median 3D...", "x=2 y=2 z=2");
+
+// Use a simple FindMaxima per Slice 
+// (more adavaned tools can be used like RS-FISH (https://github.com/PreibischLab/RS-FISH#download , update site : "Radial Symmetry") )
```
[the remark this answers is not in the document]
Comment (romainGuiet) [reply scripts/celldemo_FindMaxima_makeSpots.groovy]: fixed in 17a9d97d8a7e28c73eff34f323df17e874fa899e

File: scripts/celldemo_FindMaxima_makeSpots.groovy [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+#@Double (value = 1.0) zRad_spot
+#@Boolean (value=false ) showImages
+
+/*
+ * This scripts demoes how to :
+ * A - make use of Fiji to detect spots 
+ * B - create Imaris spots from the Results Table
+ */
+
+// start-up cleaning
```
[the remark this answers is not in the document]
Comment (romainGuiet) [reply scripts/celldemo_FindMaxima_makeSpots.groovy:15]: and the roiManager , close all the images and make sure to be connected to Imaris

File: scripts/celldemo_FindMaxima_makeSpots.groovy (hunk 1 above)
Comment (romainGuiet) [reply scripts/celldemo_FindMaxima_makeSpots.groovy]: is it more clear with d10eaff8eb3a3bbb44a8d35193359b3b9c5c3aa4  ?

File: scripts/celldemo_FindMaxima_makeSpots.groovy (hunk 2 above)
Comment (romainGuiet) [reply scripts/celldemo_FindMaxima_makeSpots.groovy:90]: see d10eaff8eb3a3bbb44a8d35193359b3b9c5c3aa4
